dist/docs/guide/channels/index.adoc

```diff
@@ -2,13 +2,88 @@
 
 More information on the wildfly channels can be found in https://github.com/wildfly-extras/wildfly-channel/blob/main/doc/spec.adoc. This chapter is intended to explain some concepts behind different channel types and how they can be used to provide software updates.
 
-### "Open" channels
+A channel contains a collections of artifact streams used by a server. Each stream should contain only backwards-compatible artifact updates with new versions of artifacts replacing previous ones.
 
-"Open" channels expose the latest available versions of artifacts found in their repositories as the component version. The versions can be curated using either version patterns or a blocklist excluding specific versions.
+There are two ways for a stream to determine the versions of its artifact:
 
-This type of channel can be useful when the content of repositories can be controlled and trusted. Any new artifact deployed into the channel repository, will automatically be made available for subscribed servers.
+ * using an artifact manifest with fixed versions, or
+ * rely on Maven metadata.
```

Comment:
The "Maven metadata" seems a bit abstract. Can it be called "using regular expression version pattern" or something similar instead?

Reply:
Or "dynamic version patterns"?

Reply:
The maven-metadata.xml is what ultimately dictates the version when either regular expressions or resolve-if-no-latest is used. The regular expressions are just used to filter subset of the available versions. 

Is one of this better "use versions from maven-metadata.xml in underlying repositories" or "use versions provided by underlying Maven repositories"?

Reply:
I wouldn't mention "maven metadata" because that's implementation detail of maven repositories. If maven repo is in consistent state the metadata files should be sorted out.

What about "use the latest artifact version available in backing Maven repository", eventually expand of version patterns in separate paragraph?